lib/entry/StateManager.js

```diff
@@ -143,14 +140,27 @@ module.exports = function(liveEntry) {
                         return updateFlavorsToServer(liveEntry.entryId, liveEntry.entryServerNodeId, liveEntry.flavorsDownloaders);
                     }
                 })
```

Comment:
you remove thie catch code which is important in case if entryServerNodeId changed  getEntryServerNodeId() will not know to re-take the  new entryServerNodeId

Reply:
my mistake. Fixed